packages/dd-trace/src/plugins/util/git-repo-info.js

```diff
@@ -0,0 +1,356 @@
+// From https://github.com/rwjblue/git-repo-info/blob/d3ab418ef8b392eabbe911a37871708b15201b70/index.js
```

Comment:
The file should also be added to `LICENSE-3rdparty.csv` similar to the TDigest entry.